.github/workflows/ci.yml

```diff
@@ -107,8 +132,7 @@ jobs:
     defaults:
       run:
         working-directory: ${{ inputs.data_project_dir }}
-    if: ${{ always() }}
-    needs: [ci_branching]
+    if: ${{ github.event.action == 'closed' }}
```

Comment:
this requires users to add the `closed` event to their ci template, otherwise it won't be executed on merge/close

Reply:
OK. Let's remember to add in our generated yml from cli